pkgs/by-name/gl/globalprotect-openconnect_2/gui.nix

```diff
@@ -0,0 +1,70 @@
+{
+  fetchurl,
+  stdenv,
+  zstd,
+  webkitgtk_4_0,
+  libappindicator,
+  gtk3,
+  cairo,
+  gdk-pixbuf,
+  libsoup,
+  glib,
+  autoPatchelfHook,
+}:
+let
+  version = "2.3.7";
```

Comment:
Can we inherit the version from the base package? To prevent someone from being able to potentially mix and match the GUI component from the rest.